main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
```

Comment:
Why does this function produce and return a list? To me it would make more sense return the same datatype (`dict`), as normalizing doesn't inherently change the structure of data.

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6